types/stripe/index.d.ts

```diff
@@ -7746,6 +7748,83 @@ declare namespace Stripe {
         }
     }
 
+    namespace oauth {
+        interface IOAuthAuthorizationCodeTokenRequest {
+            /**
+             * authorization_code when turning an authorization code into an access token
+             */
+            grant_type: 'authorization_code';
+            /**
```

Comment:
On the new interfaces there should be a newline before block comments.

eg:
```
            /**
             * authorization_code when turning an authorization code into an access token
             */
            grant_type: 'authorization_code';

            /**
             * The value of the code
             */
            code: string;
```

Reply:
Maybe we should add a prettier rule for this? e.g. https://eslint.org/docs/rules/lines-around-comment

Reply:
nvm I see you are using tslint and couldn't find any equivalent rule